x-pack/plugins/cloud_security_posture/common/constants.ts

```diff
@@ -49,3 +49,9 @@ export const CSP_RULE_TEMPLATE_SAVED_OBJECT_TYPE = 'csp-rule-template';
 
 export const CLOUDBEAT_VANILLA = 'cloudbeat/cis_k8s'; // Integration input
 export const CLOUDBEAT_EKS = 'cloudbeat/cis_eks'; // Integration input
+
+export const LOCAL_STORAGE_LATEST_FINDINGS_KEY = 'cloudPosture:latestFindings';
+export const LOCAL_STORAGE_PAGINATION_RESOURCE_FINDINGS_KEY = 'cloudPosture:resourceFindings';
+export const LOCAL_STORAGE_PAGINATION_FINDINGS_BY_RESOURCE_KEY = 'cloudPosture:findingsByResource';
+export const LOCAL_STORAGE_PAGINATION_BENCHMARK_KEY = 'cloudPosture:benchmark';
+export const LOCAL_STORAGE_PAGINATION_RULES_KEY = 'cloudPosture:rules';
```

Comment:
1. consider using `page size` instead of `pagination` so we won't confuse that with the page index
2. consider adding `page size` to the values as well, so we'll have some margin to add more local storage keys at the same pages if we'll need to.
3. `LOCAL_STORAGE_LATEST_FINDINGS_KEY` is missing the `PAGINATION` in name which you added to the other constants